loan-payment-skip/REPWRITERSPECS/BANNO.LOANPAYMENT.SKIP.V1.POW

```diff
@@ -204,7 +209,8 @@ DEFINE
 
  DEFAULTFEEAMOUNT          = $35.00
  DEFAULTLOANTRACKINGTYPE   = 77
- DEFAULTMAXSKIPS           = 2
+ DEFAULTMAXSKIPSYEAR       = 0

```

Comment:
Hi @tkainz , I request you change line 212 "DEFAULTMAXSKIPSYEAR = 0" back to a default of 2, or change the cfg to say the default is 0.  
Other than that The code looks good and I think this is a a good idea to add max skips per life of loan as we came across it as an addition CU's made in the NetTeller version of the PowerOn.

Reply:
@jordan-keenan Because of the discrepancy between the default of 2 and how the program is currently coded: The CFG "DEFAULT" value is 2, but the instructions state that a blank value (which would imply the default value) results in no limit. However, the code treated a blank value as the default of 2. The decision was made to leave the CFG Letter file parameter setting for 2 as a default in place, but set the code up such that a blank value in the CFG Letter file would indeed result in there being no limit.

Reply:
> @jordan-keenan Because of the discrepancy between the default of 2 and how the program is currently coded: The CFG "DEFAULT" value is 2, but the instructions state that a blank value (which would imply the default value) results in no limit. However, the code treated a blank value as the default of 2. The decision was made to leave the CFG Letter file parameter setting for 2 as a default in place, but set the code up such that a blank value in the CFG Letter file would indeed result in there being no limit.

OK. Thanks for explaining, Tom. It's a bit of a Funky setup, but if that is what was decided . . .